amplipi/streams.py

```diff
@@ -973,6 +973,72 @@ def info(self) -> models.SourceInfo:
     source.track = "Not currently supported"
     return source
 
+class Aux(BaseStream):
+  """ A stream to play from the aux input. """
+  def __init__(self, name: str, disabled: bool = False, mock: bool = False):
+    super().__init__('aux', name, disabled=disabled, mock=mock)
+    self.bkg_thread = None
+
+  def reconfig(self, **kwargs):
+    reconnect_needed = False
+    if 'disabled' in kwargs:
+      self.disabled = kwargs['disabled']
+    if 'name' in kwargs:
+      self.name = kwargs['name']
+    if reconnect_needed:
+      last_src = self.src
+      self.disconnect()
+      time.sleep(0.1) # delay a bit, is this needed?
+      self.connect(last_src)
+
+  def connect(self, src):
+    """ Connect a short run VLC process with audio output to audio source """
+    print(f'connecting {self.name} to {src}...')
+
+    if self.mock:
+      self._connect(src)
+      # make a thread that waits for a couple of secends and returns after setting info to stopped
+      self.bkg_thread = threading.Thread(target=self.wait_on_proc)
+      self.bkg_thread.start()
+      return
+
+    # Set input source
+    amixer_command = f'amixer -D usb71 set "PCM Capture Source" "Line"'
```

Comment:
My isntincts are that we should be using the hw:cmedia8chint audio device created by our udev rules. Our tests have a similar command string in them:
```
# Ensure the Aux Input (Line on the CMedia chip) is selected for capture.
subprocess.run(shlex.split('amixer -D hw:cmedia8chint set "PCM Capture Source",0 Line'),

```

Reply:
We should probably add a utils.py function for this, maybe something like `enable_aux_input`.  utils.py is where most of this type of functionality is kept.